### ISSUE TITLE
Migrate to develocity

### ISSUE THREAD

**Issue by ZacSweers** [cdsap/InfoKotlinProcess#25]
I see a couple of warnings that I believe are coming from this plugin

```
WARNING: The following functionality has been deprecated and will be removed in the next major release of the Develocity Gradle plugin. For assistance with migration, see https://gradle.com/help/gradle-plugin-develocity-migration.
- The deprecated "gradleEnterprise.buildScan.buildFinished" API has been replaced by "develocity.buildScan.buildFinished"
- The deprecated "gradleEnterprise.buildScan.value" API has been replaced by "develocity.buildScan.value"
```

**Comment by cdsap** [comment]
thanks for the report. Included in the new release 0.1.6 just published.
It uses the new Develocity API if the project uses Develocity plugin >= 3.17

**Comment by ZacSweers** [comment]
Thanks!
